### ISSUE TITLE
An error ocurred while starting the kernel  when  I'm running spder 

### ISSUE THREAD

**Issue by songpeng326** [spyder-ide/spyder#6164]
## Description

**What steps will reproduce the problem?**

1.
2. 
3. 

**What is the expected output? What do you see instead?**
An error ocurred while starting the kernel
Your Python environment or installation doesn't have the ipykernel module installed on it. Without this module is not possible for Spyder to create a console for you.

You can install ipykernel by running in a terminal:

pip install ipykernel

or

conda install ipykernel

but ipykernel already installed
**Please provide any additional information below**

  File "/home/lab326/anaconda3/lib/python3.5/site-packages/spyder/plugins/ipythonconsole.py", line 890, in run_script
    self.execute_code(line, current_client, clear_variables)
  File "/home/lab326/anaconda3/lib/python3.5/site-packages/spyder/plugins/ipythonconsole.py", line 936, in execute_code
    sw.execute(to_text_string(to_text_string(lines)))
  File "/home/lab326/anaconda3/lib/python3.5/site-packages/qtconsole/history_console_widget.py", line 47, in execute
    source, hidden, interactive)
  File "/home/lab326/anaconda3/lib/python3.5/site-packages/qtconsole/console_widget.py", line 631, in execute
    self._execute(source, hidden)
  File "/home/lab326/anaconda3/lib/python3.5/site-packages/qtconsole/frontend_widget.py", line 294, in _execute
    msg_id = self.kernel_client.execute(source, hidden)
AttributeError: 'NoneType' object has no attribute 'execute'

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.5.2
* Qt Versions: 5.9.3, PyQt5 5.9.2 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.3.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.1.3 (OK)
pandas >=0.13.1  :  0.18.1 (OK)
numpy >=1.7      :  1.11.3 (OK)
sphinx >=0.6.6   :  1.4.6 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.9.0 (OK)
psutil >=0.3     :  4.3.1 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
sympy >=0.7.3    :  1.0 (OK)
cython >=0.21    :  0.24.1 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
IPython >=4.0    :  5.1.0 (OK)
pylint >=0.25    :  1.5.4 (OK)

```

**Comment by ccordoba12** [comment]
This error was fixed in Spyder 3.2.4, please update.

**Comment by songpeng326** [comment]
thank you very much
